rotkehlchen/chain/evm/decoding/structures.py

```diff
@@ -76,11 +76,13 @@ class DecodingOutput:
     and is used in post-decoding rules like in the case of balancer
     - refresh_balances may be set to True if the user's on-chain balances in some protocols has
     changed (for example if the user has deposited / withdrawn funds from a curve gauge).
-    """
+    - reload_decoders can be None in which case nothing happens. Or a set of decoders names for which to reload data
+    """  # noqa: E501
```

Comment:
the set is of the class name right? This would be good info to state here. I see myself using the counterparty instead

Reply:
will do yeah